jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -160,37 +171,42 @@ protected void configure(QuickStartGeneratorConfiguration generator, WebAppConte
     @Override
     public void configure(WebAppContext context) throws Exception
     {
-        if (!_quickStart)
+        if (!isQuickStartEnabled(context))
```

Comment:
I think this change will break the other modes.

Reply:
The other modes were fine in testing.
I reworked this without this new attribute, just relying on the information from the existing attribute for mode and the existence or not of the quickstart-web.xml based on that mode.